gourmet/__init__.py

```diff
@@ -1,4 +1,5 @@
 from gi import require_version
 require_version("Gdk", "3.0")
+require_version('Gst', '1.0')
```

Comment:
A bit of a nitpick here... Could you use either double quotes or single quotes for all strings in that file?